src/components/notifications/DingDing.vue

```diff
@@ -12,5 +12,11 @@
                 <a href="https://developers.dingtalk.com/document/robots/custom-robot-access" target="_blank">https://developers.dingtalk.com/document/robots/custom-robot-access</a> <a href="https://open.dingtalk.com/document/robots/customize-robot-security-settings#title-7fs-kgs-36x" target="_blank">https://open.dingtalk.com/document/robots/customize-robot-security-settings#title-7fs-kgs-36x</a>
             </i18n-t>
         </div>
+
+        <label for="isAtAll" class="form-label">{{ $t("isAtAll") }}<span style="color: red;"><sup>*</sup></span></label>
+        <select id="isAtAll" v-model="$parent.notification.isAtAll" class="form-select" required>
+            <option value="true">{{ $t("TRUE") }}</option>
+            <option value="false">{{ $t("FALSE") }}</option>
+        </select>
```

Comment:
What I meant was something like this, but
- with the respective translations in `en.json`,
- the respective changes to `dingding.js` and
- testing that I did not make a typo in the github webeditor ^^^

What do you think about the label? Is this self-explaiantory enough? Do we need to add an helptext? (would not know what to put there, but still would like to have your oppinon on this one)


```suggestion
        <label for="metioning" class="form-label">{{ $t("Mentioning") }}<span style="color: red;"><sup>*</sup></span></label>
        <select id="metioning" v-model="$parent.notification.metioning" class="form-select" required>
            <option value="nobody">{{ $t("Don't mention people") }}</option>
            <i18n-t keypath="Mention everyone" tag="option" value="@everyone">
                <code>@everyone</code>
            </i18n-t>
        </select>
```

Reply:
And?
It does not really make sense to notify everybody and specific people. You either notify nobody, somebody or everybody.

=> a selection between these would make sense to me.

Reply:
> Can you adjust the relevant code as required?

What do you mean?
I suggested a change above.

Reply:
Please consider using selection values such as `nobody`/`everybody` which convey the meaning to the reader if they only read the backend code instead of `true`/`false`

Is there a reason you did not include the `i18n-t` part?